manifests/f/Flow-Launcher/Flow-Launcher/1.8.0/Flow-Launcher.Flow-Launcher.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Flow-Launcher.Flow-Launcher
+PackageVersion: 1.8.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/Flow-Launcher/Flow.Launcher/releases/download/v1.8.0/Flow-Launcher-v1.8.0.exe
+  InstallerSha256: 42856B9C0F2E0256EDD58E4E4886380E7D562E4CDBA172C706929D4BAC9E314D

```

Comment:
```suggestion
  InstallerSha256: 42856B9C0F2E0256EDD58E4E4886380E7D562E4CDBA172C706929D4BAC9E314D
  InstallerSwitches:
    Silent: --silent
```

Reply:
what's the change in this one?

Reply:
it does not need the silent switch, the installation is silent by default

Reply:
It opens the Application, which is not silent.

Reply:
It opens the application after the installation is done but has no pop up windows or require any user interaction during the installation.